app/models/persistent_volume_claim.rb

```diff
@@ -0,0 +1,6 @@
+class PersistentVolumeClaim < ActiveRecord::Base
```

Comment:
actually, the precise naming would be `PersistentContainerVolumeClaim`
but as `PersistentVolume` is already named wrong, this is in line with it :smile: 

I'd prefer naming this correct and putting refactoring of `PersistentVolume` to `PersitentContainerVolume` in the backlog. 
@blomquisg your call